web/config/environment.js

```diff
@@ -11,6 +11,8 @@ const getEnv = (key, defaultValue) => {
   return value != null ? value : defaultValue;
 };
 
+const hasGoogleAnalyticsID = !!getEnv("GOOGLE_ANALYTICS_ID", null);
```

Comment:
Could we use `GOOGLE_ANALYTICS_TAG_ID` to be consistent with the server config option?